packages/react-components/tests/browser/RichTextSendBox.spec.tsx

```diff
@@ -57,15 +57,15 @@ betaTest.describe('RichTextSendBox tests', () => {
     await expect(component).toHaveScreenshot('rich-text-send-box-with-system-message.png');
   });
 
-  betaTest('RichTextSendBox should show attachmentsWithProgress correctly', async ({ mount }) => {
+  betaTest('RichTextSendBox should show attachments correctly', async ({ mount }) => {
     const component = await mount(
       <RichTextSendBox
         onSendMessage={async () => {
           return;
         }}
-        attachmentsWithProgress={[
-          { progress: 0.65, id: 'id1', name: 'test1.pdf', extension: 'pdf' },
-          { progress: 1, id: 'id2', name: 'test2.docx', extension: 'docx' },
+        attachments={[
+          { progress: 0.65, id: 'id1', name: 'test1.pdf' },
+          { progress: 1, id: 'id2', name: 'test2.docx' },
```

Comment:
just a random thought, do we handle progress greater than 1.0?

Reply:
right, Fluent UI is capable of handling it, but we need to handle it at UI library, will add a logic for this, thanks

Reply:
actually we do have logic for it: 
<img width="743" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/23f2b02b-8505-456f-8e2a-0b20e973501e">

if progress is > 1, the progress bar will be set to hidden